invenio_circulation/api.py

```diff
@@ -132,6 +132,31 @@ def is_item_available_for_checkout(item_pid):
         return search_result.hits.total == 0
 
 
+def is_item_at_desk_available_for_checkout(item_pid, patron_pid):
+    """Return True if the ITEM_AT_DESK is available for loan, False instead."""
+    # Try first to accept the availability for given patron
+    except_config_exists = current_app.config.get(
```

Comment:
Can you elaborate why you need this config? I would say that this method is very specific to `ITEM_AT_DESK`, so I would rather simply:
1. move this method inside the `ItemAtDeskToItemOnLoan` class, because it is not needed in another cases
2. instead of having yet another config, which in this case you should not be allowed to change it, I would probably hardcode the state?